orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForUpInstancesTask.groovy

```diff
@@ -219,4 +240,24 @@ class WaitForUpInstancesTask extends AbstractWaitingForInstancesTask {
 
     return deployment?.capacity as Map<String, Integer>
   }
+
+  public static class Splainer {
+    List<String> messages = new ArrayList<>()
+
+    def add(String message) {
+      messages.add(message)
+      return this
+    }
+
+    def splain() {
+      log.info(messages.join("\n  - "))
```

Comment:
I think `log` exists from the enclosing classes `@Slf4j` annotation - if we were going to generalize `Splainer` to use elsewhere it would probably make sense to take in the logger name/class as a constructor arg?

Reply:
Yes!